lllc/CMakeLists.txt

```diff
@@ -1,3 +1,17 @@
 add_executable(lllc main.cpp)
 target_link_libraries(lllc PRIVATE lll)
 
+include(GNUInstallDirs)
+install(TARGETS lllc DESTINATION "${CMAKE_INSTALL_BINDIR}")
```

Comment:
Just remove `""`. They actually can corrupt paths.

Reply:
@chfast I followed the convention in [solc/CMakeLists.txt](https://github.com/ethereum/solidity/blob/develop/solc/CMakeLists.txt):
```
install(TARGETS solc DESTINATION "${CMAKE_INSTALL_BINDIR}")
```

Reply:
Yes, I was doing the same, but it turned out it does not anything. And sometimes is harmful.